.travis.yml

```diff
@@ -33,4 +33,4 @@ script:
     - $TRAVIS_BUILD_DIR/tools/execute-unit-tests-with-coverage
     - jshint static/main/js/*.js static/setup/js/*.js test/selenium/*.js test/integration/lib/*.js test/integration/test/*.js
     - npm run test-integration # starts foxbox and kills it within the test script
-    - (cargo run -- -l $BOX_LOCAL_NAME -p $BOX_PORT &> /dev/null &) ; npm run test-selenium
+    - (cargo run -- -l $BOX_LOCAL_NAME -p $BOX_PORT --disable-tls &> /dev/null &) ; npm run test-selenium
```

Comment:
@samgiles Just run selenium with a Firefox profile that has the security exception for the certificate you're using in the tests, see e.g. https://blog.codecentric.de/en/2010/02/selenium-and-ssl-certificates/.

I don't agree we don't lose anything - currently, we have a http API _with_ a test. Once we merge this, we will have a https API _without_ a test. :(

If some upstream change in Iron causes our API server to break on https but not on http, we will not be able to see that in our tests. Right?

Can you create a follow-up ticket to repair the selenium test?